Levels/Level.gd

```diff
@@ -152,7 +155,7 @@ func spawn_drone(spawn_pos: Vector3, owner: Node) -> void:
 	enemy_drone.nav = self
 	enemy_drone.replan_actions()
 	enemy_drone.player = player
-	enemy_drone.translation = get_closest_point(Vector3(spawn_pos.x + 1, spawn_pos.y + 0.5, spawn_pos.z + 1))
+	enemy_drone.translation = get_closest_point(Vector3(spawn_pos.x + 1, 0, spawn_pos.z + 1))
```

Comment:
Does this cause problems if the drone is spawned somewhere with a very high or low elevation (y translation different from 0)?